pipeline_games/load_games.py

```diff
@@ -0,0 +1,256 @@
+"""Script for loading to database"""
```

Comment:
General Praise: This file looks very fiddly and complex! Good work on getting it all done!